generator/internal/genclient/translator/protobuf/protobuf.go

```diff
@@ -190,24 +171,25 @@ func normalizeTypes(in *descriptorpb.FieldDescriptorProto, field *genclient.Fiel
 	}
 }
 
-func (t *Translator) processMessage(m *descriptorpb.DescriptorProto, mFQN string, parent *genclient.Message) *genclient.Message {
-	message := &genclient.Message{}
-	t.state.MessageByID[mFQN] = message
-	message.Name = m.GetName()
-	message.ID = mFQN
-	message.Parent = parent
+func processMessage(state *genclient.APIState, m *descriptorpb.DescriptorProto, mFQN string, parent *genclient.Message) *genclient.Message {
```

Comment:
I don't claim to know what is idiomatic in Go, but this is basically making the functions methods on the `state` parameter...

Reply:
I wanted to do away with the `Translator` struct, since it didn't seem to be adding anything and removing made things easier to reason about